test/integration-tests/Assets.test.ts

```diff
@@ -110,6 +110,22 @@ describe('Assets e2e Testing', () => {
     expect(numberResults === null ? false : Number(numberResults) > 0)
   })
 
+
+  test('Search for an asset by query', async () => {
+
+    const query = "{\"bool\":{\"must\":[{\"query_string\":{\"query\":\"**\",\"fields\":[\"service.attributes.main.name\"]}}]}}"
+
+    const queryAssetCommand = `${baseCommands.assets.queryAsset} ${query} `
+    console.debug(`COMMAND: ${queryAssetCommand}`)
+
+    const searchStdout = execCommand(queryAssetCommand, execOpts)
+    console.log(`STDOUT: ${searchStdout}`)
+
+    const numberResults = parseNumberResultsFromSearch(searchStdout)
+    console.log(`Number of Results: ${numberResults}`)
+    expect(numberResults === null ? false : Number(numberResults) > 0)
```

Comment:
this only tests that the command is executed but not if it's working correctly

Reply:
fixed now

Reply:
I mean we should do a real query and expect that a result is returned

Reply:
ok, searching by attribute name now and checking that results > 0